services/QuillLMS/app/helpers/pages_helper.rb

```diff
@@ -143,6 +143,11 @@ def team_info
             title: 'Professional Learning Manager',
             img: 'team-sherry-lewkowicz@2x.png'
           },
+          {
+            name: 'Charlie Looper',
+            title: 'Partnerhips Specialist II',
+            img: 'team-charlie-looper@2.png'
```

Comment:
Do we want '2x.png' so it is consistent with other image names?

Reply:
I did this because I uploaded a wrongly-formatted image of the same name, and our cloudfront instance has no expiry and no asset versioning. However, I believe I've found a way to invalidate the CDN at asset-level granularity. If that works, I'll change the file name.

Reply:
Update: cache has been invalidated and PR updated with standardized file paths.